src/__tests__/RenderParentDashboard.test.js

```diff
@@ -33,26 +34,61 @@ const Component = props => {
 };
 
 describe('<ParentDashboard /> test suite', () => {
-  test('Welcome Back', () => {
-    render(<Component parent={{ children: [] }} />);
-    expect(screen.getByText(/Welcome Back/i)).toBeInTheDocument();
-  });
-
   test('Dashboard Nav', () => {
     render(<Component parent={{ children: [] }} />);
-    expect(screen.getByText(/Dashboard/i)).toBeInTheDocument();
+
+    const hamburgerBtn = screen.getByRole('button', { class: 'ant-btn' });
+    fireEvent(
```

Comment:
The drawer for the nav only appears in the document flow after clicking the hamburger button. This being so, it was necessary to totally rewrite the parent dashboard test suite.